### PR TITLE
Add a silly test for RevWalk count

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Comment by kshyatt** [comment]
What's the deal with that AV fail?

**Comment by tkelman** [comment]
Something is trying to delete files without closing them first. Since it's a new failure, probably one of the recently-added tests.

**Comment by kshyatt** [comment]
Alright, now it's appearing on both AV runs. I'll investigate tonight.

**Comment by tkelman** [comment]
if this failure isn't happening on master, then guess it is related to the change here

**Comment by kshyatt** [comment]
The beautiful 💚  of passing AV tests.

File: test/libgit2.jl [hunk 1]
```diff
@@ -831,6 +831,9 @@ mktempdir() do dir
                 for i in eachindex(oids)
                     @test cache_oids[i] == test_oids[i]
                 end
+                LibGit2.with(LibGit2.GitRevWalker(repo)) do walker
+                    @test count((oid,repo)->(oid == commit_oid1), walker, oid=commit_oid1, by=LibGit2.Consts.SORT_TIME) == 1
```
Comment (tkelman) [inline test/libgit2.jl]: I'd maybe line wrap this, otherwise lgtm

Comment (kshyatt) [inline test/libgit2.jl]: I might do a cleanup PR of this file soon so I'll leave that for the future.

**Pull request merged**